app/templates/components/public/speaker-item.hbs

```diff
@@ -48,19 +48,21 @@
             {{#if session.startsAt}}
               {{moment-format session.startsAt 'DD MMM YY'}}
             {{/if}}
-            <br>
-            <span class="session-location">
-              <strong>
-                {{session.microlocation.name}}
-              </strong>
-            </span>
+            {{#if session.microlocation}}
+              <br>
+              <span class="session-location">
+                <strong>
+                  {{concat (t 'Microlocation - ') session.microlocation.name}}
+                </strong>
+              </span>
+            {{/if}}
           </p>
-          <div class="ui fluid padded" style={{-html-safe (concat "color: " session.track.fontColor "px; background-color: " session.track.color ";")}}>
+          <div class="ui fluid padded" style={{concat "color: " session.track.fontColor "px; background-color: " session.track.color ";"}}>
```

Comment:
Please use the `css` helper instead of `concat`

Reply:
@niranjan94 I've updated the helper. Please review :)